clang-tools-extra/docs/clang-tidy/checks/misc/must-use.rst

```diff
@@ -0,0 +1,28 @@
+.. title:: clang-tidy - misc-must-use
+
+misc-must-use
+=============
+
+Allow strictly enforcing that variables are used for specific classes,
+even with they would not be normally warned using `-Wunused-variable` due 
+templates or custom destructors.
+
+In the following example, `future2` normally would not trigger any unused variable checks,
+but using `{key: "misc-must-use.Types", value: "std::future"}` would cause `future2` to have
```

Comment:
this configuration is in deprecated style, use new one misc-must-use.Types: 'std::future'

Reply:
Thanks TIL, fixed.